docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst

```diff
@@ -58,6 +58,13 @@ Use comments
 Even with task names and explicit state, sometimes a part of a playbook or role (or inventory/variable file) needs more explanation.
 Adding a comment (any line starting with ``#``) helps others (and possibly yourself in future) understand what a play or task (or variable setting) does, how it does it, and why.
 
+Use fully qualified collection names
+------------------------------------
+
+Use `fully qualified collection names (FQCN) <https://docs.ansible.com/ansible/latest/reference_appendices/glossary.html#term-Fully-Qualified-Collection-Name-FQCN>`_ to avoid ambiguity in which collection to search for the correct module or plugin for each task.
+
+For `builtin modules and plugins <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html#plugin-index>`_, use the ``ansible.builtin`` FQCN, for example, ``ansible.builtin.copy``. 
```

Comment:
`ansible.builtin` is not an FQCN, but a collection name.

Reply:
@felixfontein i thought `ansible.builtin` is fully qualified **collection** name
and `ansible.builtin.copy` is a module name which uses FQCN as its prefix.

If i'm wrong, please suggest changes

Reply:
to me it sounds like fully qualified name of a collection and if it's applied to a module name, it sounds a bit confusing

Reply:
Ok, I'll change it to match the definition from glossary and let's discuss this later

Reply:
thanks!

Reply:
https://github.com/ansible/ansible-documentation/pull/155

Reply:
When I wrote that comment suggestions didn't work (for whatever random quirk of the GH UI), otherwise I would have created one :)